### PR TITLE
Clone ref if `ref` setting is set

### PR DESCRIPTION
**Pull request by qwerty287**
Closes https://github.com/woodpecker-ci/plugin-git/issues/81

### CODE REVIEW

**Comment by xoxys** [comment]
What is the benefit?

File: flags.go [hunk 1]
```diff
@@ -25,13 +25,12 @@ var globalFlags = []cli.Flag{
 	&cli.StringFlag{
 		Name:    "sha",
 		Usage:   "git commit sha",
-		EnvVars: []string{"PLUGIN_SHA", "CI_COMMIT_SHA"},
+		EnvVars: []string{"CI_COMMIT_SHA"},
```
Comment (anbraten) [inline flags.go:28]: Shouldn't we still support sha cloning in case someone wants to do it manually?

**Review by anbraten — COMMENTED** [review]
Not sure if I understand the changes in here. You want to clone using `ref` as soon as its set. But by default WP would still clone using sha?

File: flags.go (hunk 1 above)
Comment (anbraten) [reply flags.go:28]: Would leave it to not risk being breaking.

Comment (anbraten) [reply flags.go:28]:
```suggestion
		EnvVars: []string{"PLUGIN_SHA", "CI_COMMIT_SHA"},
```

**Review by anbraten — APPROVED** [review]

**Pull request merged**